appveyor.yml

```diff
@@ -1,3 +1,19 @@
+# The config expects the following environment variables to be set:
+#  - "GN_CONFIG" Build type. One of {'debug', 'testing', 'release'}.
+#  - "GN_EXTRA_ARGS" Additional gn arguments for a build config,
+#      e.g. 'fatal_linker_warnings=false enable_linux_installer=false'.
```

Comment:
Given that appveyor is Windows only maybe use Windows example, eg ```target_cpu = \"x86\"``` for 32 bit builds

Reply:
Done